WcaOnRails/app/views/competitions/_competition_form_new.html.erb

```diff
@@ -0,0 +1,26 @@
+<% is_actually_confirmed = @competition.persisted? ? Competition.find(@competition.id).confirmed? : false %>
+<% mail_to_wcat = mail_to Team.wcat.email, t("competitions.competition_form.wcat"), target: "_blank" %>
+<%= react_component("CompetitionForm", {
+    competition: {
```

Comment:
is there a reason you can't do `competition: @competition`?

Reply:
I was going to write a specific serializable_hash for it, but as Comptitions already had one and I didn't want to break anything (yet) I did it this way. I plan to move it either to the serializable_hash method or a seperate one in the competitions controller once all the properties are filled out